src/app/qt/facefilter/CMakeLists.txt

```diff
@@ -209,27 +211,30 @@ else()
   endif()
 
   ### Workaround for Linux's linker problems {
-  set(fix_cyclic FALSE)
-  if(APPLE)
-    set(fix_cyclic FALSE)
-  elseif(CMAKE_COMPILER_IS_GNUCXX)
-    # MinGW included
-    set(fix_cyclic TRUE)
-  elseif(NOT UNIX)
-    set(fix_cyclic FALSE)
-  elseif("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
-    # Linux Clang
-    set(fix_cyclic TRUE)
-  endif()
 
-  if(fix_cyclic)
-    string(
+  if(NOT ANDROID)
```

Comment:
We are not building executable on Android so `if(NOT ANDROID)` not needed. `fix_cyclic` do no harm.